astropy/coordinates/tests/test_representation.py

```diff
@@ -1026,11 +1026,13 @@ def __new__(cls, angle, unit=None, wrap_angle=180 * u.deg, **kwargs):
                                    **kwargs)
             return self
 
-    class SphericalWrap180Representation(SphericalRepresentation):
-        attr_classes = OrderedDict([('lon', Longitude180),
-                                    ('lat', Latitude),
-                                    ('distance', u.Quantity)])
-        recommended_units = {'lon': u.deg, 'lat': u.deg}
+    with pytest.warns(AstropyDeprecationWarning,
+                      match="'recommended_units' attribute is deprecated"):
+        class SphericalWrap180Representation(SphericalRepresentation):
+            attr_classes = OrderedDict([('lon', Longitude180),
+                                        ('lat', Latitude),
+                                        ('distance', u.Quantity)])
+            recommended_units = {'lon': u.deg, 'lat': u.deg}
```

Comment:
p.s. Why are the tests using a *deprecated* feature?

Reply:
I think you are right - we should just remove the `recommended_units` property here.  (At first I thought this was due to me, as I remember wanting to ensure that if someone wrote a representation subclass prior to when I removed `recommended_units`, they would get a warning that they should remove it. But that is done further down.)